### PR TITLE
update md

### PR DESCRIPTION
**Pull request by zcohn**

### CODE REVIEW

File: recommendations/SourceCodeManagement.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
+# Souce Code Management
```
Comment (Seanstoppable) [inline recommendations/SourceCodeManagement.md]: Source vs Souce

Comment (zcohn) [inline recommendations/SourceCodeManagement.md]: fixed, thanks

**Comment by Seanstoppable** [comment]
:+1:

File: recommendations/Databases.md [hunk 2]
```diff
@@ -5,14 +5,13 @@
   - [Aster](http://www.asterdata.com)
     - for data warehouse
   - [DynamoDB](http://aws.amazon.com/dynamodb/)
-  - [ElastiCache](http://aws.amazon.com/elasticache/)
+  - [ElastiCache](http://aws.amazon.com/elasticache/) (preference to [Redis](http://redis.io/))
+    - As a caching layer, **Adopt**.
+    - As a primary data store, **Hold**.
```
Comment (andreyk0) [inline recommendations/Databases.md]: It's a cache:) Why would anybody even think about using it as a primary data store?

**Comment by andreyk0** [comment]
:+1: LGTM overall, one small comment inline

**Pull request merged**